FerramAerospaceResearch/LEGACYferram4/FARWingAerodynamicModel.cs

```diff
@@ -161,16 +161,12 @@ public class FARWingAerodynamicModel : FARBaseAerodynamics, IRescalable<FARWingA
 
         private FARWingAerodynamicModel parentWing;
         private bool updateMassNextFrame;
-        [KSPField(isPersistant = true)] private float? massOverride;
+        [KSPField(isPersistant = false)] public float massOverride = float.MinValue;
```

Comment:
Keep the member private, it is already exposed through a property, it should probably stay `isPersistent = true` since it won't be modified by the module itself

Reply:
For some reason it has to be public to let the .cfg file set it. I can double check that that was indeed needed.

Reply:
I double checked and the member does indeed have to be public to be able to read the .cfg file value